email-client.js

```diff
@@ -41,12 +41,14 @@ exports.generateParams = function(userName, data) {
                 Data: 'UWATERLOO DATA'
             }
         },
-        Source: `Conrad Progress Report from xuan <x34ren@edu.uwaterloo.ca>`
-        //Source: 'Email from Xuan <'+recipient+'>'
+        Source: 'uwalexacoop@gmail.com',
+        ReturnPathArn: 'arn:aws:ses:us-east-1:528823259307:identity/uwalexacoop@gmail.com',
```

Comment:
What is arn for? Does this mean it sends the email to uwalexacoop email?

Reply:
It's just to ensure that the email being sent is being sent to the uwalexacoop email